apps/src/templates/instructions/codeReviewV2/CodeReviewTimeline.jsx

```diff
@@ -0,0 +1,86 @@
+import React, {useEffect, useRef} from 'react';
+import PropTypes from 'prop-types';
+import CodeReviewTimelineElement, {
+  codeReviewTimelineElementType
+} from '@cdo/apps/templates/instructions/codeReviewV2/CodeReviewTimelineElement';
+import CodeReviewTimelineCommit from '@cdo/apps/templates/instructions/codeReviewV2/CodeReviewTimelineCommit';
+import CodeReviewTimelineReview from '@cdo/apps/templates/instructions/codeReviewV2/CodeReviewTimelineReview';
+import {
+  commitShape,
+  reviewShape
+} from '@cdo/apps/templates/instructions/codeReviewV2/shapes';
+
+const dataType = {
+  review: 'review',
+  commit: 'commit'
+};
+
+const CodeReviewTimeline = props => {
+  const {reviewData, commitsData} = props;
+
+  const timelineEndRef = useRef(null);
+
+  const scrollToBottom = () => {
+    timelineEndRef.current?.scrollIntoView({behavior: 'smooth'});
+  };
+
+  useEffect(() => {
+    scrollToBottom();
+  }, [reviewData, commitsData]);
```

Comment:
Does this typically only happen once (after the data arrives)?  (I could imagine this being disorienting if it happens more often.)

Reply:
Yeah the intent is that this just happens when the data loads. This will probably be easier to tweak when things are more wired up and I'm able to walk through the different flows